lib/src/ui/pages/news_list/news_list_page.dart

```diff
@@ -69,7 +69,7 @@ class _NewsPageState extends State<NewsListPage> {
             ),
           ),
           PrimaryTextButton(
-            text: Strings.showAll,
+            text: Strings.of(context).showAll,
```

Comment:
przy zmianie na polski to się rozjeżdża 
![image](https://user-images.githubusercontent.com/24854180/82524690-d951b400-9b2f-11ea-827c-2427c19c0b66.png)

Reply:
true i task na dostosowanie tego ekranu jest osobno ;)